Jenkinsfile

```diff
@@ -6,8 +6,21 @@ elifePipeline {
     }
 
     node('containers-jenkins-plugin') {
-        stage 'Build images', {
+        stage 'Build critical CSS images', {
             checkout scm
+            sh "find build/critical-css -name '*.css' -type f -delete"
+            sh "touch build/critical-css/default.css"
+            sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d --build critical_css"
```

Comment:
Using `up` rather than `build` to just build the dependencies for `critical_css` rather than everything (though it is most of them).

Reply:
Hmm, is actually all of them!